content/webapp/utils/iiif/v3/index.ts

```diff
@@ -581,7 +581,7 @@ export function getCollectionManifests(
           return getCollectionManifests(item);
         }
       })
-      .flat(Infinity);
+      .flat(Infinity) as CollectionItems[];
```

Comment:
Flagged as a TS issue. I _think_ this is right, but maybe @gestchild you could confirm?

Reply:
looks right to me